.github/workflows/openapi-docs-pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Test OpenAPI doc is valid on Pull Requests
+
+on:
+  pull_request:
+    paths:
+      - "schemas/openapi.yml"
+jobs:
+  build:
+    name: Run Spectral
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: stoplightio/spectral-action@v0.8.0
+        with:
+          file_glob: 'schemas/openapi.yml'
```

Comment:
Are we thinking we'll have an overall OpenAPI specification, or one per set of API versions?

Reply:
separate - I've moved it into the v1alpha directory